tests/Unit/EndpointTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Makeable\ApiEndpoints\Tests\Unit;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Http\Request;
+use Makeable\ApiEndpoints\Endpoint;
+use Makeable\ApiEndpoints\Tests\Stubs\Server;
+use Makeable\ApiEndpoints\Tests\Stubs\User;
+use Makeable\ApiEndpoints\Tests\UnitTestCase;
+
+class EndpointTest extends UnitTestCase
+{
+    use RefreshDatabase;
```

Comment:
I could be wrong, but I believe this test case doesn't actually hit the database. It's basically just at unit test.

However I think we should have a separate TestCase that actually *does* use the Laravel http testing api to hit a test-controller and see data returned from the database. If nothing else, to serve as a documentation for how to use the package.